erts/emulator/beam/erl_message.h

```diff
@@ -291,12 +291,23 @@ typedef struct {
     ErtsMessage **last; /* ... last (non-message) signal */
 } ErtsMsgQNMSigs;
 
+/*
+ * The ErtsRecvMarker struct is used for two other types of markers
+ * namely yield markers and prio queue markers.
+ */
+#define ERTS_RECV_MARKER_TYPE_RECV              0
```

Comment:
An enum?

Reply:
Trying to save space in the receive markers. The type of the field is an `unsigned char`.

Reply:
Apparently in C23 they have added support in enum to do this, that is `enum erts_recv_marker:signed char { ... };`, but until we get to use that this will have to do.

Reply:
Nice!